.github/ISSUE_TEMPLATE/bug.yml

```
@@ -22,6 +22,8 @@ body:
           required: true
         - label: I have resolved all warnings from `brew doctor` and that did not fix my problem.
           required: true
+        - label: I have done a [search for similar issues](https://github.com/Homebrew/homebrew-core/issues).
```

Mmm, the markdown link doesn't seem to work: https://github.com/Homebrew/homebrew-core/blob/46c70a3dcb9d2291d79adc0c9ab9a4b9d7b77141/.github/ISSUE_TEMPLATE/bug.yml

```suggestion
        - label: I have done a search for similar issues (https://github.com/Homebrew/homebrew-core/issues).
```